scripts/watch.js

```diff
@@ -0,0 +1,63 @@
+const fs = require( "fs" );
+const http = require( "http" );
+const path = require( "path" );
+
+const rollup = require( "rollup" );
+const loadConfigFile = require( "rollup/dist/loadConfigFile" );
+
+const server = http.createServer( ( req, res ) => {
+	try {
+		const requestedPath = path.format( {
+			root: process.cwd(),
+
+			// path.normalize will remove any "..".
+			// Probably useless as at least Chrome resolves any ".."
+			// _before_ sending the request.
+			// Probably still a good idea to avoid accessing anything
+			// _outside_ of process.cwd() anyways.
+			base: path.normalize( req.url )
+		} );
+		if ( fs.existsSync( requestedPath ) ) {
+			return res.end( fs.readFileSync( requestedPath ) );
+		}
+		res.statusCode = 404;
+		res.end( `Not found: ${req.url}.` );
+	} catch ( e ) {
+		res.statusCode = 500;
+		res.end( `Unexpected error: ${e.message}.` );
+	}
+} );
+
+server.listen( 4000 );
+
+loadConfigFile( path.resolve( process.cwd(), "rollup.config.js" ) ).then(
+	async( { options, warnings } ) => {
+
+		console.log( `We currently have ${warnings.count} warnings` );
+		warnings.flush();
+
+		for ( const optionsObj of options ) {
+			const bundle = await rollup.rollup( optionsObj );
+			await Promise.all( optionsObj.output.map( bundle.write ) );
+		}
```

Comment:
I got this entire piece of code straight out of the [`rollup` docs](https://rollupjs.org/guide/en/#programmatically-loading-a-config-file).
I tried with and without this `for` and it seems to work just the same. Perhaps one of the reviewers with more experience on `rollup` can comment?

Reply:
I think this loop is for if you have multiple bundle inputs/outputs, but we currently have just the one. LGTM.

Reply:
Thanks for looking into this. I'll leave this as-is unless you have another preference.